manifests/nexus.pp

```diff
@@ -44,6 +44,8 @@
   Optional[String]  $proxy_server    = undef,
   Optional[String]  $proxy_type      = undef,
   Optional[Boolean] $allow_insecure  = undef,
+  Optional[String]  $proxy_type      = undef,
```

Comment:
you're repeating a param that's there, two lines above

Reply:
Sorry, my fault. I should be more careful. I have removed the duplicate parameter